src/components/BlockCard/BlockCard.tsx

```diff
@@ -10,7 +10,7 @@ interface IProps {
 
 export default function BlockCard(props: IProps) {
   const { block } = props;
-  const d = moment(new Date(parseInt(block.timestamp, 16) * 1000).toISOString()).format('MMMM Do YYYY, h:mm:ss a')
+  const d = moment(new Date(parseInt(block.timestamp, 16) * 1000).toISOString()).format("MMMM Do YYYY, h:mm:ss a");
```

Comment:
what is the Do vs DD ?

Reply:
i think it adds the postfix `th`

![image](https://user-images.githubusercontent.com/364566/62656511-2e159c80-b919-11e9-87c4-7008e114d2b3.png)